facebook/OpenBIC
# common: sensor: Modify mp5990 device init config

Summary:
- Skip setting iout_cal_gain/iout_oc_fault_limit if given **0xFFFF** in mp5990_init_args.
- In Waimea Canyon, Cause of iout_cal_gain already set in chip's register as default value. So modify mp5990_init_args's iout_cal_gain to 0xFFFF.

Test Plan:
- Build Code: PASS
- Check sensor reading **MB HSC Input Pwr** should greater than **MB CPU PWR** via BMC console: PASS

Log:
- BMC console:
  ```
  root@bmc-oob:~# sensor-util mb 0x60
  mb:
  MB CPU PWR                   (0x60) :   50.00 Watts | (ok)
  root@bmc-oob:~# sensor-util mb 0x61
  mb:
  MB HSC Input Pwr             (0x61) :   72.00 Watts | (ok)
  ```
